src/Graph.elm

```diff
@@ -243,19 +206,22 @@ computeEdgeDiff old new =
     let
         collectUpdates edgeUpdate updatedId label =
             let
-                replaceUpdate old =
-                    case ( old, edgeUpdate label ) of
+                replaceUpdate old_ =
+                    case ( old_, edgeUpdate label ) of
                         ( Just (Remove oldLbl), Insert newLbl ) ->
                             if oldLbl == newLbl then
                                 Nothing
+
                             else
                                 Just (Insert newLbl)
 
                         ( Just (Remove _), Remove _ ) ->
-                            Debug.crash "Graph.computeEdgeDiff: Collected two removals for the same edge. This is an error in the implementation of Graph and you should file a bug report!"
+                            -- Debug.crash "Graph.computeEdgeDiff: Collected two removals for the same edge. This is an error in the implementation of Graph and you should file a bug report!"
+                            Nothing
```

Comment:
Not sure if Nothing is the best default (here and below).

Reply:
The cases that would lead to a crash should return `Just eu`, where `eu` is the the value of `edgeUpdate label`. You could probably merge that with the last case and just have that case match on `(_, eu)` instead of `(Nothing, eu)`. This is strictly worse than what we had before, but at least it's compatible with Elm 0.19.

Reply:
Should this go through the crashLoop too?